src/components/TryItOutButton/styles.module.css

```diff
@@ -1,11 +1,17 @@
 
-.tryOutWrapper .btnCancel {
-    color: var(--cancel-button-color);
-    border-color: var(--cancel-button-border-color);
+.tryOutWrapper .btnTryOut {
+    color: var(--try-out-button-color)!important;
+    border-color: var(--try-out-button-border-color)!important;
 }
 
-.tryOutWrapper .btnTryOut {
-    color: var(--try-out-button-color);
-    border-color: var(--try-out-button-border-color);
+.tryOutWrapper .btnTryOut:hover {
+    box-shadow: unset;
+    border-color: var(--KButtonSecondaryHoverBorder, var(--black-45, rgba(0, 0, 0, 0.45)));
```

Comment:
Please do no use and overwrite kongponents variables

If needed - please create a new variable and set value to corresponding value